src/pages/Proposal.vue

```diff
@@ -237,6 +250,29 @@ export default defineComponent({
 		toggleModalVisible() {
 			this.isModalVisible = !this.isModalVisible;
 		},
+		//  execute_before must be after voting_end
```

Comment:
Instead of doing this, can we disable all the day and time on or before voting_end and just allow the selection date ahead of voting_end?

Reply:
I think that would be less intuitive for user, since he wouldn't understand why the dates are disabled, also what if the user add execute input before the voting input, it wouldn't work that way